### PR TITLE
Add GetTotalExpenditure to UG

### PR DESCRIPTION
**Pull request by woojiahao**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1401,10 +1401,213 @@ Command Options: This command does not take in any arguments and will not proces
...
+
+Command: `get_total_expenditure month/Month [c/Category] [year/Year]`
+
+Command Options:
+
+| Option Name | Optional? | Purpose                                                                               |
+|-------------|-----------|---------------------------------------------------------------------------------------|
+| month/      | No        | Month to calculate the total expenditure.                                             |
+| c/          | Yes       | Category of expenditure to retrieve.<br>Defaults to all categories if not provided.   |
+| year/       | Yes       | Year to calculate the total expenditure.<br>Defaults to current year if not provided. |
```
Comment (lipwei1808) [inline docs/UserGuide.md:1423]: Want to mention it cateogyr is case insensitive?

**Review by lipwei1808 — APPROVED** [review]
LGTM, great work on so many example cases! Just one small question but else is good to merge

File: docs/UserGuide.md (hunk 1 above)
Comment (woojiahao) [reply docs/UserGuide.md:1423]: Sure

**Review by lipwei1808 — APPROVED** [review]
lezgeddismerged

**Pull request merged**